Sources/ClientRuntime/Config/DefaultSDKRuntimeConfiguration.swift

```diff
@@ -6,12 +6,25 @@
 //
 
 public struct DefaultSDKRuntimeConfiguration: SDKRuntimeConfiguration {
```

Comment:
We should rename this to something more descriptive and without the terns "Default" and "SDK".
I'll file an issue to address this